### ISSUE TITLE
Renderer.prototype.table: <tbody> text included in rendered output

### ISSUE THREAD

**Issue by eezing** [mikaelbr/marked-terminal#315]
### Description

```gherkin
Given source text includes a valid Markdown table
When calling parse with source text
Then rendered table includes <tbody> in cell 1
And has extra row at bottom 
```

### Environment
- Node.js v20.12.0
- marked v13.0.3
- marked-terminal v7.4.0

### Example

**Code:**
```ts
import { Marked } from 'marked';
import { markedTerminal } from 'marked-terminal';

const body = `
| Character Name | Hometown        | Description                               |
|----------------|-----------------|-------------------------------------------|
| Aria Windwalker| Elmridge        | A swift and agile archer from the woods   |
| Thorne Ironfist| Stonekeep       | A strong warrior from a mountain fortress |
| Lyra Moonshade | Sunhaven        | A cunning sorceress from a desert city    |
`;

// @ts-ignore
const m = new Marked(markedTerminal());

console.log(m.parse(body));
```

**Console:**
![Screenshot 2024-09-18 at 17 38 34](https://github.com/user-attachments/assets/37cd7ee2-9829-4a11-8ea5-13e3faef1fd4)

### Possible Cause
https://github.com/mikaelbr/marked-terminal/blob/e8f60dd8a95291f2e58d6b1e43a0848719a53562/index.js#L231

Removing this line seemed to resolve the issues.

**Comment by kabeep** [comment]
Same problem and 100% reproduction.

### Environment

node.js@18.18.0
├── marked-terminal@7.1.0
└── marked@14.1.2

**Comment by mikaelbr** [comment]
Should be fixed in `marked-terminal@7.2.1`
